accounts/admin.py

```diff
@@ -155,91 +108,145 @@ def clean_username(self):
+                                 'anonymised account' % username)
             return HttpResponseRedirect(reverse('admin:auth_user_changelist'))
-        info = obj.profile.get_info_before_delete_user(remove_sounds=True)
-        model_count = {model._meta.verbose_name_plural: len(objs) for model,
-                objs in info['deleted'].model_objs.items()}
-        tvars = {'anonymised': []}
-        anon = {}
-        anon['model_count'] = dict(model_count).items()
-        anon['logic_deleted'] = info['logic_deleted']
-        anon['name'] = info['anonymised']
-        tvars['anonymised'].append(anon)
+
+        user_info = obj.profile.get_info_before_delete_user(include_sounds=True, include_other_related_objects=False)
+        user_info['deleted_objects_details'] = {}
+        model_count = {model._meta.verbose_name_plural: len(objs) for
+                       model, objs in user_info['deleted'].model_objs.items()}
+        user_info['deleted_objects_details']['model_count'] = dict(model_count).items()
+        user_info['deleted_objects_details']['logic_deleted'] = user_info['logic_deleted']
+
+        tvars = {'users_to_delete': [], 'type': 'delete_include_sounds'}
+        tvars['users_to_delete'].append(user_info)
+
         return render(request, 'accounts/delete_confirmation.html', tvars)
 
-    delete_include_sounds.label = "Soft delete user (delete sounds)"
-    delete_include_sounds.short_description = disable_active_user.short_description
+    delete_include_sounds.label = "Delete user and sounds"
+    delete_include_sounds.short_description = "Delete the user and the sounds"
 
-    def delete_preserve_sounds(self, request, obj):
+    def delete_spammer(self, request, obj):
         username = obj.username
         if request.method == "POST":
+            logger.info('Requested async deletion of user {0} - {1}'.format(obj.id,
+                                                                            DELETE_SPAMMER_USER_ACTION_NAME))
             gm_client = gearman.GearmanClient(settings.GEARMAN_JOB_SERVERS)
             gm_client.submit_job("delete_user",
-                    json.dumps({'user_id': obj.id, 'action': DELETE_USER_KEEP_SOUNDS_ACTION_NAME}),
-                wait_until_complete=False, background=True)
+                                 json.dumps({'user_id': obj.id, 'action': DELETE_SPAMMER_USER_ACTION_NAME}),
+                                 wait_until_complete=False, background=True)
             messages.add_message(request, messages.INFO,
-                                 'User \'%s\' will be soft deleted asynchronously. Comments and other content '
-                                 'will appear under anonymised account' % username)
+                                 'User \'%s\' will be deleted asynchronously including sounds and all of its related'
+                                 'content.' % username)
```

Comment:
double-check here. because you're using the split line strings, you need the space between the words related and content

Reply:
Addressed in fa0c6c256904c9de2d60b7f7beba8081da961e5f